plugins/toolkit/resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -761,6 +761,7 @@ codewhisperer.codescan.run_scan_complete=<html><body> Security Scan completed fo
 codewhisperer.codescan.run_scan_error=Amazon Q encountered an error while scanning for security issues. Please try again later.
 codewhisperer.codescan.run_scan_error_telemetry=Security scan failed.
 codewhisperer.codescan.run_scan_info=Select 'Run' in toolbar to scan this package for security issues.
+codewhisperer.codescan.run_scan_throttling_error_telemetry= Too many requests, please wait before trying again.
```

Comment:
Is this error user facing or is this error being sent to telemetry?

Reply:
No this is sent to telemetry not customer facing message.

Reply:
This should probably not reside here then cause this file has user-facing text. Telemetry reason could be moved within the code to where we are sending the telemetry or a utils file.

Reply:
Moved this text from `MessageBundle.properties` to the respective file.